src/math/Triangle.js

```diff
@@ -183,6 +183,65 @@ THREE.Triangle.prototype = {
 
 	},
 
+	closestPointToPoint: function () {
+
+		var plane, edgeList, projectedPoint, closestPoint;
+
+		return function closestPointToPoint( point, optionalTarget ) {
+
+			if ( plane === undefined ) plane = new THREE.Plane();
+			if ( edgeList === undefined ) edgeList = [ new THREE.Line3(), new THREE.Line3(), new THREE.Line3() ];
+			if ( projectedPoint === undefined ) projectedPoint = new THREE.Vector3();
+			if ( closestPoint === undefined ) closestPoint = new THREE.Vector3();
+
+			var result = optionalTarget || new THREE.Vector3();
+			var minDistance = Infinity;
+
+			// project the point onto the plane of the triangle
+
+			plane.setFromCoplanarPoints( this.a, this.b, this.c );
+			plane.projectPoint( point, projectedPoint );
+
+			// check if the projection lies within the triangle
+
+			if( this.containsPoint( projectedPoint ) === true ) {
```

Comment:
That's the basic question. It was more of a general thought - independent of where it gets implemented. Also figured, in case you pick up the barycentric suggestion, you may end up not calling this routine.

Let's see what @WestLangley thinks about it...